wooey/backend/utils.py

```diff
@@ -422,16 +440,16 @@ def test_fastx(filepath):
 
 def create_job_fileinfo(job):
     parameters = job.get_parameters()
-    from ..models import WooeyFile
+    from ..models import WooeyFile, UserFile
```

Comment:
Yeah. So the new model is you have two files: WooeyFile is the 'system file' that is unique. The UserFile points to the WooeyFile and essentially grants them access to it. This is also nice because it naturally opens up things like file sharing between users (grant access to user X for this file, and can maybe even go to a Group)